Tzkt.Api/Websocket/Processors/OperationsProcessor.cs

```diff
@@ -439,28 +453,14 @@ public async Task Subscribe(IClientProxy client, string connectionId, Operations
                 #region add to subs
                 foreach (var type in parameter.TypesList)
                 {
-                    if (!TypesSubs.TryGetValue(type, out var typeSub))
-                    {
-                        typeSub = new();
-                        TypesSubs.Add(type, typeSub);
+                    if (parameter.Entrypoints?.Count != 0) {
+                        // parameter.TypesList == ["transactions"] — verified by `parameter.EnsureValid()`
+                        foreach (var entrypoint in parameter.Entrypoints)
+                            TryAdd(EntrypointsSubs, entrypoint, parameter.Address, connectionId);
                     }
-                    if (parameter.Address == null)
-                    {
-                        typeSub.All ??= new();
-                        if (typeSub.All.Add(connectionId))
-                            Limits[connectionId] = Limits.GetValueOrDefault(connectionId) + 1;
-                    }   
                     else
                     {
-                        typeSub.Addresses ??= new(4);
-                        if (!typeSub.Addresses.TryGetValue(parameter.Address, out var addressSub))
-                        {
-                            addressSub = new(4);
-                            typeSub.Addresses.Add(parameter.Address, addressSub);
-                        }
-
-                        if (addressSub.Add(connectionId))
-                            Limits[connectionId] = Limits.GetValueOrDefault(connectionId) + 1;
+                        TryAdd(TypesSubs, type, parameter.Address, connectionId);
```

Comment:
Since we allow entrypoints with multiple op types specified, we should do this:
```cs
foreach (var type in parameter.TypesList)
{
    if (parameter.Entrypoints != null && type == Operations.Transactions)
    {
        foreach (var entrypoint in parameter.Entrypoints)
            TryAdd(EntrypointsSubs, entrypoint, parameter.Address, connectionId);
    }
    else
    {
        TryAdd(TypesSubs, type, parameter.Address, connectionId);
    }
}
```